src/execution_plan.c

```diff
@@ -1507,17 +1524,13 @@ static void ExecutionPlan_RegisterForRun(ExecutionPlan* ep){
 
 void FlatExecutionPlan_AddToRegisterDict(FlatExecutionPlan* fep){
     Gears_dictAdd(epData.registeredFepDict, fep->id, fep);
-    // call the on registered callback if set
-    if(fep->onRegisteredStep.stepName){
-        RedisGears_FlatExecutionOnRegisteredCallback onRegistered = FlatExecutionOnRegisteredsMgmt_Get(fep->onRegisteredStep.stepName);
-        RedisModule_Assert(onRegistered);
-        onRegistered(fep, fep->onRegisteredStep.arg.stepArg);
-    }
+    EPTurnOnFlag(fep, FEFRegistered);
```

Comment:
This macro is for ExecutionPlan, We should create one from FlatExecutionPlan or change the name (remove the EP prefix)

Reply:
added ones for FEP - copy paste for now, but if we need changes it is nice to have